sdk/src/main/java/com/horizen/backwardtransfer/BackwardTransferFunctionsImpl.java

```diff
@@ -0,0 +1,52 @@
+package com.horizen.backwardtransfer;
+
+import com.horizen.box.WithdrawalRequestBox;
+import org.bouncycastle.pqc.math.linearalgebra.ByteUtils;
+
+import java.util.EnumMap;
+import java.util.List;
+
+
+public class BackwardTransferFunctionsImpl implements BackwardTransferFunctions {
```

Comment:
What do you think, if we structure zendo_sc_crypto_lib logic into interfaces grouped by crypto type.
For example, "SchnorrFunctions", "VrfFunctions", "PoseidonFunctions", ZeroKnowledgeProofsFunctions", etc.
Or just have a single one "ZendooCryptoUtils".